src/js/dropdown-menu/dropdown-menu-view.js

```diff
@@ -76,6 +76,7 @@ define([
                 this.$menu = this.$('.dropdown-menu');
                 this.$page = $(document);
                 this.$dropdownButton = this.$('.js-dropdown-button');
+                this.$lastItem = this.$menu.find('li:last-child a');
```

Comment:
There was an issue when I wasn't calling event.preventDefault() on a tab + shift that made it jump to the penultimate ```li``` and I didn't realize this was the issue for a while. I do think the current way could be a little more performant 'tho because only creating the jQuery element once as opposed to every time it is needed.